config/install/field.field.paragraph.lesson_section.field_lesson_section_content.yml

```diff
@@ -6,6 +6,7 @@ dependencies:
     - paragraphs.paragraphs_type.lesson_checklist
     - paragraphs.paragraphs_type.lesson_divider
     - paragraphs.paragraphs_type.lesson_embedded_video
+    - paragraphs.paragraphs_type.lesson_footnotes
```

Comment:
@adubovskoy the configs are not installable - you forgot to export the footnotes paragraph itself.

Reply:
@spleshka pushed, tnx, But there are 2 issues with anu_module itself:
1. On my local copy, on admin/config/development/features/edit/anu_lms , the export path is always 'modules/contrib' and I can't change it, so, I have to copy by hand every time when change config.
2. it would be nice to be able to test the complete installation cycle in CI or `make` command, could help to avoid mistakes like that.

Reply:
1. Let's look at this on the call together. I don't have this problem locally.
2. This is true. We're working on it already, you can follow the updates in https://github.com/systemseed/anu_lms/pull/113